src/Shared/RedisUtility.cs

```diff
@@ -75,17 +75,25 @@ internal static object GetObjectFromBytes(byte[] dataAsBytes)
                 return null;
             }
 
-            BinaryFormatter binaryFormatter = new BinaryFormatter();
-            using (MemoryStream memoryStream = new MemoryStream(dataAsBytes, 0, dataAsBytes.Length))
+            try
             {
-                memoryStream.Seek(0, System.IO.SeekOrigin.Begin);
-                object retObject = (object)binaryFormatter.Deserialize(memoryStream);
-
-                if (retObject.GetType() == typeof(RedisNull))
+                BinaryFormatter binaryFormatter = new BinaryFormatter();
+                using (MemoryStream memoryStream = new MemoryStream(dataAsBytes, 0, dataAsBytes.Length))
                 {
-                    return null;
+                    memoryStream.Seek(0, System.IO.SeekOrigin.Begin);
+                    object retObject = (object)binaryFormatter.Deserialize(memoryStream);
+
+                    if (retObject.GetType() == typeof(RedisNull))
+                    {
+                        return null;
+                    }
+                    return retObject;
                 }
-                return retObject;
+            }
+            catch (Exception ex)
+            {
+                LogUtility.LogError("Error GetObjectFromBytes in RedisUtility:", ex.Message, ex.InnerException, ex.StackTrace);
```

Comment:
Its not the correctly written: First parameter to any log method is string format and as in above message you are having three parameters after that so string format should have place holder like {0},{1} etc for that.

Also, Instead of heaving ex.Message, ex.InnerException and ex.StackTrace use just ex.ToString(). It will do same for you.

Basically use: LogUtility.LogError("GetObjectFromBytes in RedisUtility:{0}", ex.ToString());